### PR TITLE
1. Get GCN code working on toy dataset

### PR DESCRIPTION
**Pull request by RussH-code**
Added comments to the original Ramirez et.al code to increase readability and aid understanding.

Added generate_toy_data.py to generate expression and adjacency matrix.
Added GCN_test with functions to use the GCN code with numpy array and pandas dataframe data structures

Code was able to run and training results were generated.

### CODE REVIEW

File: GCN_Cancer/GCN_Cancer/PPI.py [hunk 1]
```diff
@@ -107,24 +110,23 @@
...
-if False:
-    grid_params = {}
-    data = (train_data, train_labels, val_data, val_labels, test_data, test_labels)
-    utils.grid_search(params, grid_params, *data, model=lambda x: models.cgcnn(L,**x))
+common['F']              = [1]         # Number of graph convolutional filters.
+common['K']              = [1]         # Polynomial orders.
+common['p']              = [2]         # Pooling sizes.
+common['M']              = [1024, C]   # Output dimensionality of fully connected layers.
+
+# if True:
```
Comment (ipoole) [inline GCN_Cancer/GCN_Cancer/PPI.py:118]: Is this commented out code needed?  Can it be removed?

File: GCN_Cancer/GCN_Cancer/GCN_test.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+"""
+Written by RH
+"""
+import sys
+sys.path.insert(0, '..')
```
Comment (ipoole) [inline GCN_Cancer/GCN_Cancer/GCN_test.py:5]: This is not nice!   I managed to get the code working without this, and lib recognised, by setting the second level GCN_Cancer folder (!) to "Source root" in PyCharm.   Please try this aiming to remove the sys.path.insert(...)

**Review by ipoole — CHANGES_REQUESTED** [review]
I was able to run the code - generate_toy_data and GCN_Test.  Please consider the couple of comments I've made.

File: GCN_Cancer/GCN_Cancer/GCN_test.py (hunk 2 above)
Comment (RussH-code) [reply GCN_Cancer/GCN_Cancer/GCN_test.py]: Removed sys.path.insert(...)

**Review by ipoole — APPROVED** [review]
Thanks!

**Comment by ipoole** [comment]
Resolves #1

**Pull request merged**